### PR TITLE
Docs 1.17

### PR DESCRIPTION
**Pull request by daminisatya**
Testing

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -261,6 +261,7 @@ different Kubernetes components.
 | `VolumeScheduling` | `true` | Beta | 1.10 | 1.12 |
 | `VolumeScheduling` | `true` | GA | 1.13 | - |
 | `VolumeSubpath` | `true` | GA | 1.13 | - |
+| `VolumeSubpathEnvExpansion` | `true` | GA | 1.17 | - |
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: @daminisatya , I think during the revert I removed the alpha and beta additions of this feature in the second table.
Lines 153-154 should be removed from the first table and added to the second table before line 264.

**Comment by jimangel** [comment]
We force updated `dev-1.17` with this PR'd commit to fix the conflicts, removing the need for this PR.
/close

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (jimangel) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: This might need to be fixed via a PR to `dev-1.17`

Comment (daminisatya) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: +1

**Pull request closed without merging**